base/managers.jl

```diff
@@ -48,6 +48,65 @@ end
 # to be mutually reachable without a tunnel, as is often the case in a cluster.
 # Default value of kw arg max_parallel is the default value of MaxStartups in sshd_config
 # A machine is either a <hostname> or a tuple of (<hostname>, count)
+"""
+    addprocs(machines; tunnel=false, sshflags=\`\`, max_parallel=10, kwargs...) -> List of process identifiers
+
+Add processes on remote machines via SSH. Requires `julia` to be installed in the same
+location on each node, or to be available via a shared file system.
+
+`machines` is a vector of machine specifications. Workers are started for each specification.
+
+A machine specification is either a string `machine_spec` or a tuple - `(machine_spec, count)`.
+
+`machine_spec` is a string of the form `[user@]host[:port] [bind_addr[:port]]`. `user` defaults
+to current user, `port` to the standard ssh port. If `[bind_addr[:port]]` is specified, other
+workers will connect to this worker at the specified `bind_addr` and `port`.
+
+`count` is the number of workers to be launched on the specified host. If specified as `:auto`
+it will launch as many workers as the number of cores on the specific host.
+
+Keyword arguments:
+
+* `tunnel`: if `true` then SSH tunneling will be used to connect to the worker from the
+            master process. Default is `false`.
+
+* `sshflags`: specifies additional ssh options, e.g.::
+    sshflags=`-i /home/foo/bar.pem`
```

Comment:
This should remain a fenced code block lined up with the first backtick in `` `sshflags` ``, the `::` is rst syntax. This will be what is causing the formatting issue @tkelman noticed [further down](https://github.com/JuliaLang/julia/pull/17688/files/bda169900daae27e974c36c577d0d4f7dd0d9f36..1d54068e2ff4fc70a9c09be82c181a2b8548fb8f#r72743252).